### DISCUSSION TITLE
The concerning target platform coverage of Visual Studio Code

### DISCUSSION THREAD

**Discussion by martin-braun** [unoplatform/uno#8902] [General]
I'm a Mac user who uses Visual Studio Code and wants to dive into Uno soon. As many of you, I am seeking for the ultimate high language grail which engraves "write once, compile for everything" on its front. A system to target all platforms without messing with low-level to work highly efficient. I don't care if it is C# or TS, but I'm not a fan of NPM and Deno is still too far behind, so Uno is probably what I am looking for. Avalonia uses a canvas on the web target, while Uno renders HTML elements if I am not mistaken. I prefer the latter and I think it's a better choice to peek interests of people who want to go web-first. Something not even Flutter does.

However, for me it is a huge bummer that Visual Studio Code is unable to compile for Android, iOS and macOS (Xamarin) when JetBrains Raider is. I would use Raider, if it was not wasting my battery so heavily (sorry JetBrains team, your Java foundation is an energy waster). 

Flutter is capable to compile and debug on Android and iOS with the right tools installed, so I was wondering what are the current limits of Uno to get the same level of convenience for Code users?

**Comment by jeromelaban** [comment]
Support for iOS and Android is on the roadmap, but we're not there yet. You can use Visual Studio 2022 for Mac for now.

**Reply by martin-braun** [reply]
Thank you, I know I can and I probably have to. I just want to point out that I avoided Visual Studio for Mac so far, successfully. Code has all the extensions I like and love and is my absolute favorite choice, because it is light-weight, especially since I disable extensions that I didn't use in a while every now and then. Please keep this discussion informed, I would switch to Code as soon as it's possible. Or how could I get notified? I haven't found an issue I can subscribe to. Thanks.
